Datadog/Example/ExampleTests/ExampleTests.swift

```diff
@@ -0,0 +1,40 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import XCTest
+@testable import Example
+
+
+class ExampleTests: XCTestCase {
```

Comment:
I'm not sure if having tests target in `Example` app is the best choice. As this app is used by the SDK unit tests (as a host), integration tests (as a subject) and benchmarks (host), we may need to modify it in this way or another either for the SDK or Test Runner. This may not play well in the future.

How about creating separate `TestRunner` app with `TestRunnerTests`? If we do so, would it be additionally possible to insturment it with the `HTTPServerMock` (Python server) to add integration tests for the test runner? Right now the `DDTestRunner` has no tests coverage, but it runs pretty important business logic. I think it would be a good subject for the integration testing.

Reply:
Yes, that is a great idea, will work on it.